examples/complete/variables.tf

```diff
@@ -22,6 +22,17 @@ variable "region" {
   default     = "eu-gb"
 }
 
+variable "cross_region_location" {
+  description = "Specify the cross-regional bucket location. Supported values are 'us', 'eu', and 'ap'. If you pass a value for this, do not pass one for var.region"
```

Comment:
"If you pass a value for this, do not pass one for var.region"

^^ Remove this from the example variable description. In our example, we will need both variables to be passed, as region is used to provision other things such as COS instance, key protect and observability instances